torch/fx/experimental/proxy_tensor.py

```diff
@@ -1594,14 +1594,14 @@ def __init__(self, scope_root: GraphModule) -> None:
         tracer = self
 
         class AttrProxy(_AttrProxy):
-            def __init__(self, base: Module, path: str) -> None:
+            def __init__(self, base: Module, path: str, **kwargs) -> None:
                 # Class is modified to be a subclass of torch.nn.Module
                 # Warning: We blow away our own attributes here to mimic the base class
                 # - so don't expect `self.x` to do anything useful.
                 self.__class__ = type(
                     base.__class__.__name__,
                     (self.__class__, base.__class__),
-                    {},
+                    **kwargs,
```

Comment:
I think this is incorrect. I assume you meant:
```suggestion
                    {},
                    **kwargs,
```
since the third argument is the new class dict - but I assume you want to pass these as kwargs to the constructed `__init_subclass__`.